packages/mui-joy/src/MenuItem/MenuItem.test.tsx

```diff
@@ -184,6 +184,20 @@ describe('Joy <MenuItem />', () => {
     expect(menuitem).to.have.attribute('aria-disabled', 'true');
   });
 
+  it('should add external id to DOM and override internal id', () => {
```

Comment:
This is passing in master too but i just wanted to add a test which tests adding extrenal id to DOM when available